environment/environment.go

```diff
@@ -805,31 +771,31 @@ func (m *Environment) findPodCountInDeploymentManifest() int {
 			continue
 		}
 		for _, j := range *json {
-			m := j.(map[string]interface{})
+			m := j.(map[string]any)
 			// if the kind is a deployment then we want to see if it has replicas to count towards the app count
 			kind := m["kind"].(string)
 			if kind == "Deployment" || kind == "StatefulSet" {
-				podCount += getReplicaCount(m["spec"].(map[string]interface{}))
+				podCount += getReplicaCount(m["spec"].(map[string]any))
```

Comment:
nit: better to have a check for the presence of "spec" key

Reply:
Updated